packages/discord.js-utilities/src/lib/MessagePrompter.ts

```diff
@@ -0,0 +1,381 @@
+import type {
+	Message,
+	User,
+	TextChannel,
+	NewsChannel,
+	APIMessageContentResolvable,
+	MessageOptions,
+	MessageAdditions,
+	DMChannel,
+	CollectorFilter,
+	EmojiIdentifierResolvable,
+	MessageReaction,
+	EmojiResolvable,
+	GuildEmoji,
+	ReactionEmoji
+} from 'discord.js';
+
+/**
+ * This is a [[MessagePrompter]], a utility to send a message with Yes/No promt.
```

Comment:
```suggestion
 * This is a [[MessagePrompter]], a utility to send a message with Yes/No prompt.
```

Reply:
I think this statement needs an update overall, it's more than yes/no now (:

Reply:
Will update in a few, to explain more about the `MessagePrompter`